app/views/govuk_publishing_components/components/feedback/_yes_no_banner.html.erb

```diff
@@ -21,12 +21,12 @@
             <% end %>
           </li>
           <li class="gem-c-feedback__option-list-item">
-            <button class="govuk-button gem-c-feedback__prompt-link js-page-is-useful" data-track-category="yesNoFeedbackForm" data-track-action="ffYesClick" data-ga4='{"event_name":"form_submit","type":"feedback","text":"Yes", "section": "Is this page useful?"}'>
+            <button class="govuk-button gem-c-feedback__prompt-link js-page-is-useful" data-track-category="yesNoFeedbackForm" data-track-action="ffYesClick" data-ga4='{"event_name":"form_submit","type":"feedback","text":"<%= t("components.feedback.yes") %>", "section": "<%= t("components.feedback.is_this_page_useful") %>"}'>
```

Comment:
See previous comment about translations (this is why "Yes" and "Is this page useful?" was hardcoded)